rocks/designate-consolidated/rockcraft.yaml

```diff
@@ -0,0 +1,53 @@
+name: designate-consolidated
+summary: Openstack designate-consolidated
+license: Apache-2.0
+description: |
+  Ubuntu distribution of OpenStack designate-consolidated
+version: "2023.1"
+
+base: ubuntu:22.04
+platforms:
+  amd64:
+
+package-repositories:
+  - type: apt
+    cloud: antelope
+    priority: always
+
+parts:
+  designate-user:
+    plugin: nil
+    # 42411:42411 for kolla compatibility
+    overlay-script: |
+      groupadd --root $CRAFT_OVERLAY --gid 42411 --system designate
+      useradd \
+        --gid 42411 \
+        --uid 42411 \
+        --no-create-home \
+        --home /var/lib/designate \
+        --root $CRAFT_OVERLAY \
+        --system \
+        --shell /bin/false \
+        designate
+
+  designate-consolidated:
+    after: [designate-user]
+    plugin: nil
+    overlay-packages:
+      - sudo
+      - apache2
+      - libapache2-mod-wsgi-py3
```

Comment:
This package is not listed in designate-api.

Reply:
Fixed